packages/fxa-auth-server/lib/routes/subscriptions/stripe.ts

```diff
@@ -495,15 +500,19 @@ export class StripeHandler {
       request.auth,
       true
     );
+    await this.customs.check(request, email, 'createSubscriptionWithPMI');
 
     try {
-      await this.customs.check(request, email, 'createSubscriptionWithPMI');
-
-      const customer = await this.stripeHelper.fetchCustomer(uid);
+      const customer = await this.stripeHelper.fetchCustomer(uid, ['tax']);
```

Comment:
Do we always want to expand the `tax` field here even if the feature flag is disabled?

Reply:
It's not relevant, but it doesn't break anything either. I'm trying to reduce the amount of logic we need to remove later.